### DISCUSSION TITLE
How to remote debug Advice?

### DISCUSSION THREAD

**Discussion by YANGJINJUE** [open-telemetry/opentelemetry-java-instrumentation#11819] [Q&A]
How to remote debug Advice?

**Comment by steverao** [comment]
> How to remote debug Advice?

You can refer to https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/contributing/debugging.md

**Reply by trask** [reply]
hopefully this will be easier in the future after #8999 is finished

> * Allows instrumentations to have breakpoints set in them and be debugged using standard debugging techniques
> * No shading required, allowing standard debugging in IDEs
